build/performance-test.sh

```diff
@@ -70,6 +70,9 @@ cat performance-test-variable.txt
 
 printf "\nStart testing."
 ./runScenario.sh performance-test-variable.txt
+
+kubectl delete -f performance-test-fleet.yaml
```

Comment:
Is there a difference between this command and `kubectl delete fleet performance-test-fleet`?

Reply:
No difference :)

Reply:
`-f` means to read from a file so it will delete whatever resources are in the file.